### ISSUE TITLE
Escape Jinja2-like blocks

### ISSUE THREAD

**Issue by ian-r-rose** [CityOfLosAngeles/intake-dcat#21]
Some DCAT descriptions can have Jinja2-like text in them (e.g. `{{ text }}`), which intake will try to template, and then fail. We should escape those strings, or otherwise figure out how to tell intake not to attempt to template those.

**Comment by martindurant** [comment]
I would have thought Intake should ignore variables it doesn't know - perhaps needs a flag to jinja when called from Intake.

**Comment by ian-r-rose** [comment]
Here is a MWE:
```python
import intake_dcat
cat = intake_dcat.DCATCatalog('https://data.chhs.ca.gov/data.json')
entry = cat['8101cd65-eb40-40d3-8adf-23ba05d53ce8']
entry.description
```

**Comment by martindurant** [comment]
Yep, I see it. Obviously, it would be easier for me if dcat dealt with it :)

**Comment by ian-r-rose** [comment]
I'm happy to deal with it, as soon as I figure out a solution that I am happy with :)
